packages/admin-app/src/components/ElectionSetupWizard/Steps/ManifestPreviewStep.tsx

```diff
@@ -99,32 +94,13 @@ const ManifestPreviewStep: React.FC<ManifestPreviewStepProps> = ({
                             <TableCell className={classes.property}>
                                 <FormattedMessage
                                     id={MessageId.ElectionSetupManifestPreviewPropertyEndDate}
-                                    defaultMessage="End Date"
                                 />
                             </TableCell>
                             <TableCell>
                                 {preview.endDate.toLocaleDateString()}{' '}
                                 {preview.endDate.toLocaleTimeString()}
                             </TableCell>
                         </TableRow>
-                        <TableRow>
-                            <TableCell className={classes.property}>
-                                <FormattedMessage
-                                    id={MessageId.ElectionSetupManifestPreviewPropertyFileHash}
-                                    defaultMessage="File Hash"
-                                />
-                            </TableCell>
-                            <TableCell>{preview.fileHash}</TableCell>
-                        </TableRow>
-                        <TableRow>
-                            <TableCell className={classes.property}>
-                                <FormattedMessage
-                                    id={MessageId.ElectionSetupManifestPreviewPropertyFileName}
-                                    defaultMessage="File Name"
```

Comment:
File name and hash seem unimportant to me so I cut them.  If anyone feels strongly about them I can put them back in.